docs/document/docs/components/desktop/user-org-group-select.md

```diff
@@ -0,0 +1,105 @@
+---
+id: user-org-group-select
+title: UserOrgGroupSelect
+sidebar_label: UserOrgGroupSelect
+---
+
+## Overview
+The UserOrgGroup component allows the user to select user/org/group.
+
+import { UserOrgGroupSelectComponent } from "@site/static/js/samples/desktop/user-org-group-select.jsx"
+
+<UserOrgGroupSelectComponent />
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| error | string | ""  | Text to be displayed in error | Error will not be displayed if unspecified or empty |
+| icon | string | "user" | Picker icon type | The icon property is used to set the main icon displayed on the right side of the toggle part.<br/>Displays different icons based on different values of the icon property.<br/>Available options:<li>"user" : ![user](/img/icon-user.png)</li><li>"org" : ![org](/img/icon-org.png)</li><li>"group" : ![group](/img/icon-group.png)</li><br/>If the value is not specified, the default icon is "user". |
+| id | string | ""  | Component id name | |
+| label | string | ""  | Label for the component | Label is not displayed if unspecified or empty |
+| placeholder | string | "" | Placeholder text displayed in the input field | |
+| disabled | boolean | false | Enable/Disable the component | |
+| requiredIcon | boolean | false | Show/Hide the required icon | |
+| visible | boolean | true | Show/Hide the component | |
+| items | Array\<[Item](#item)\> | []  | List of options to display | Will result an error if the value of items is not an array |
+| value |  Array\<string\> | [] | Selected value | No option will be selected if the `value` are unspecified<br/>Will result an error if the value is not an array |
+
+### Item
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| label | string | null  | Text label for each option | If `Item.label` is unspecified, the value of `Item.value` is displayed on the UI |
+| type | string | null | Icon type of each item | The Item.type property determines the small icon type displayed for each option in both the dropdown menu and the selected items list.<br/>Displays different icons based on different values of the icon property.<br/>Available options:<li>"user" : ![user](/img/selected-user.png)</li><li>"org" : ![org](/img/selected-org.png)</li><li>"group" : ![group](/img/selected-group.png)</li><li>"" : No icon</li> |
```

Comment:
Removed . at the end of the sentence.
Please unify with "each option".
I suggest "toggle menu" instead of "dropdown menu" because you used it on "icon" prop description.

```suggestion
| type | string | null | Icon type of each option | The Item.type property determines the small icon type displayed for each option in both the toggle menu and the selected items list<br/>Displays different icons based on different values of the icon property<br/>Available options:<li>"user" : ![user](/img/selected-user.png)</li><li>"org" : ![org](/img/selected-org.png)</li><li>"group" : ![group](/img/selected-group.png)</li><li>"" : No icon</li> |
```